internal/runners/manifest/version.go

```diff
@@ -0,0 +1,42 @@
+package manifest
+
+import (
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/pkg/platform/api/buildplanner/model"
+	platformModel "github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/platform/runtime/artifact"
+)
+
+type resolvedVersion struct {
+	Requested string `json:"requested"`
```

Comment:
Currently with JSON output we render "Auto" as the value for Requested. For structured this should just be empty, as "Auto" is only meant for making it humanly readable.